lib/main/main.dart

```diff
@@ -181,38 +178,17 @@ class _MyHomePageState extends State<MyHomePage> {
   }
 
   Future _handleSignIn() async {
+
     final SharedPreferences prefs = await sharedPreferences;
```

Comment:
Remove prefs handling and clearing user data from from UI. Can we abstract it to some util class or function?